<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <layout class="QFormLayout" name="formLayout_10">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="Line" name="line">
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Obsługa Tabel W Bazie Danych</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_table">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <item>
               <property name="text">
                <string>Wezwanie</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Przestępstwo</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Sąd</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Uczestnik Przestępstwa</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Kara</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Więzienie</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Spis Ludności</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Świadkowie</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Dowody</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_input">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Wprowadź dane</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_save">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Zapisz dane</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_delete">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Usuń wiersz</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Specjalne Zapytania</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_specialQ">
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Ilość powiązanych przestępstw z zgłoszeniem</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Przestępstwa bez znalezionego sprawcy</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Baza przestępców oraz ilość popełnionych przez nich przestępstw</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Baza osób przebywających w więzieniu</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Baza przestępców, którzy nie dostali karę więzienia</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Kary wydane za popełnione przestępstwa</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Spisani przestepcy oraz przebieg ich sprawy w sadzie</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Postępowanie w toku</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Zeznania Świadków</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Przestępstwo połączone z rozprawą sądową</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Ilość Popełnionych Przestępstw</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Przez Osobę (Podaj Id Osoby)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>Otrzymaj Ilość Przestępstw</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="spinBox"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line_8">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Sprawdź Rozprawę Sądową Osoby</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>             Imię                                 Nazwisko</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_9">
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QLineEdit" name="lineEdit"/>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_2">
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Otrzymaj Rozprawę</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_5">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Tryb tabeli:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_tablestretch">
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Dopasowany do zawartości</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Interaktywny</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Wypełniający okno</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QPushButton" name="pushButton_submit">
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Wyślij zapytanie</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="pushButton_clearConsole">
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Czyść</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
      <item row="1" column="0">
       <widget class="QPlainTextEdit" name="queryInput">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>100</height>
         </size>
        </property>
        <property name="backgroundVisible">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>SELECT * FROM customer;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTextBrowser" name="appOstream">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZaloguj_Si"/>
   <addaction name="actionWyloguj_Si"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionZaloguj_Si">
   <property name="text">
    <string>Zaloguj Się</string>
   </property>
  </action>
  <action name="actionWyloguj_Si">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wyloguj Się</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
